bindings/node.js/src/kzg.cxx

```diff
@@ -6,6 +6,22 @@
 #include "c_kzg_4844.h"
 #include "blst.h"
 
+/**
+ * Convert C_KZG_RET to a string representation for error messages 
+*/
+std::string from_c_kzg_ret(C_KZG_RET ret) {
+    switch (ret) {
+      case C_KZG_RET::C_KZG_OK:
+        return "C_KZG_OK";
+      case C_KZG_RET::C_KZG_BADARGS:
+        return "C_KZG_BADARGS";
+      case C_KZG_RET::C_KZG_ERROR:
+        return "C_KZG_ERROR";
+      case C_KZG_RET::C_KZG_MALLOC:
+        return "C_KZG_MALLOC";
```

Comment:
There is a "not all control paths return a value" lint error for windows.  All options are covered but there is no "default".  Should I return one of the values by default to remove the lint error?

Reply:
Sure, a default case is fine.

Reply:
Which would you like returned for default, `C_KZG_ERROR`?

Reply:
Hmm good question. Is it possible to throw an exception?

If not, I think I'd return something like "UNKNOWN (27)" where "27" is integer return value.

Reply:
Ah that's right. Sounds good.